src/cljs/athens/views/filesystem.cljs

```diff
@@ -38,6 +44,40 @@
                      [:.MuiTabs-indicator {:background-color "var(--link-color)"}]]})
 
 
+(def picker-style
+  {:display         "grid"
+   :grid-auto-flow "column"
+   :grid-auto-columns "1fr"
+   :border-radius "0.5rem"
+   :flex "0 0 auto"
+   :font-size "1em"
+   :margin "0.25rem 0"
+   :align-self "stretch"
+   :overflow "hidden"
+   :transition "box-shadow 0.2s ease, filter 0.2s ease;"
+   :background (color :background-color)
+   :padding "1px"
+   ::stylefy/manual [[:&:hover {}]
+                     [:button {:text-align "center"
+                               :appearance "none"
+                               :border "0"
+                               :border-radius "calc(0.5rem - 1px)"
+                               :padding "0.5rem 0.5rem"
+                               :color "inherit"
+                               :display "flex"
+                               :justify-content "center"
+                               :align-items "center"
+                               :position "relative"
+                               :z-index "0"
+                               :background "inherit"}
+                      [:svg {:margin-inline-end "0.25em" :font-size "1.25em"}]
+                      [:&:hover {:filter "contrast(105%)"}]
```

Comment:
There's an effect when hovering over an already active tab, but no effect when hovering over an inactive tab (including no `cursor: pointer`). Should there be one?

Reply:
There is some styling, but it can be clearer. I want to make a more focused pass on all the control components to make sure they all have all the right styles, and creating additional use cases like this picker will help do that.